sdk/python/packages/flet-core/src/flet_core/video.py

```diff
@@ -178,6 +187,9 @@ def before_update(self):
         if isinstance(self.__subtitle_configuration, VideoSubtitleConfiguration):
             self._set_attr_json("subtitleConfiguration", self.__subtitle_configuration)
 
+        if isinstance(self.__configuration, VideoConfiguration):
+            self._set_attr_json("configuration", self.configuration)
```

Comment:
Could you fix that to `self.__configuration` as well. Sorry for the late reply.

Reply:
🫣 - Thanks!